identity/app/controllers/SaveContentController.scala

```diff
@@ -65,7 +61,7 @@ class SaveContentController @Inject() ( api: IdApiClient,
             if (!prefs.contains(shortUrl)) {
               val articleId = idRequest.articleId match {
                 case Some(id) => id
-                case _ =>  new URL(returnUrl).getPath.drop(1)
+                case _ => new URL(returnUrl).getPath.drop(1)
```

Comment:
Never use `URL`, always `URI`. `URL` is evil. It does weird unexpected side effects like DNS lookups!